src/app/datagrid/detail/detail.html

```diff
@@ -28,8 +28,9 @@ <h3>Simple API</h3>
   <clr-dg-row
     *clrDgItems="let user of users"
     [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.